### ISSUE TITLE
"terrascan" cannot be opened because the developer cannot be verified.

### ISSUE THREAD

**Issue by 111a5ab1** [tenable/terrascan#633]
* terrascan version: 1.4.0
* Operating System: macOS Big Sur 11.2.2

### Description

The `terrascan` Darwin binary is not signed, resulting it in its execution being blocked by default on macOS Big Sur "`because it is not from an identified developer`".

This can be bypassed and the binary _can_ be allowed to run by going into  > System Preferences > Security & Privacy > General and clicking the "`Allow Anyway`" button next to `"terrascan" was blocked from use because it is not from an identified developer`" message. 

**However**, I would expect a security tool released by a security company for macOS ("Darwin") to be signed. It would be great to see the officially released Darwin binaries signed by accuris.

### What I Did

```
./terrascan
```

**Comment by dev-gaur** [comment]
Putting this on hold until https://github.com/accurics/terrascan/issues/884 gets done. Right now, due to CGO specific complications, I'm not able to cross compile and get the binaries signed within the same GH action job.

With CGO disabled, I think It will be a more straightforward task.
